yt_dlp/extractor/vidyard.py

```diff
@@ -0,0 +1,256 @@
+                        'title': 'Install the Track for Palm Beach Polysatin Shutters With BiFold Track',
+                        'thumbnail': 'https://cdn.vidyard.com/thumbnails/5861090/RwG2VaTylUa6KhSTED1r1Q_small.png',
+                        'duration': 94,
+                    },
+                    {
+                        'id': 'ueXUJagJso7KO_tnBkYo2Q',
+                        'display_id': '41976364',
+                        'ext': 'mp4',
+                        'title': 'Install the Panel for Palm Beach Polysatin Shutters With BiFold Track',
+                        'thumbnail': 'https://cdn.vidyard.com/thumbnails/5860926/JIOaJR08dM4QgXi_iQ2zGA_small.png',
+                        'duration': 191,
+                    },
+                    {
+                        'id': 'U9hEsKlpPTMasqEf5n0KQg',
+                        'display_id': '41976382',
+                        'ext': 'mp4',
+                        'title': 'Adjust the Panels for Palm Beach Polysatin Shutters With BiFold Track',
+                        'thumbnail': 'https://cdn.vidyard.com/thumbnails/5860687/CwHxBv4UudAhOh43FVB4tw_small.png',
+                        'duration': 138,
+                    },
+                    {
+                        'id': 'gBdBzCvGXK5j2f20lAF-vg',
+                        'display_id': '41976409',
+                        'ext': 'mp4',
+                        'title': 'Assemble and Install the Valance for Palm Beach Polysatin Shutters With BiFold Track',
+                        'thumbnail': 'https://cdn.vidyard.com/thumbnails/5861425/0y68qlMU4O5VKU7bJ8i_AA_small.png',
+                        'duration': 148,
+                    },
+                ],
+            },
+            'playlist_count': 6,
+        },
+        {
+            # URL of iframe embed src
+            'url': 'https://play.vidyard.com/iDqTwWGrd36vaLuaCY3nTs.html',
+            'info_dict': {
+                'id': 'lrYeWCqR3UCAzwqUAgegHg',
+                'display_id': '9281009',
+                'ext': 'mp4',
+                'title': 'Lightbox Embed',
+                'thumbnail': 'https://cdn.vidyard.com/thumbnails/spacer.gif',
+                'duration': 39,
+            },
+        },
+    ]
+    _EMBED_REGEX = [
+        # iframe embed
+        r'<iframe[^>]+?src=(["\'])(?P<url>(?:https?:)?//play\.vidyard\.com/[\w-]+.\w+)\1',
+    ]
+    _WEBPAGE_TESTS = [
+        {
+            # URL containing inline/lightbox embedded video
+            'url': 'https://resources.altium.com/p/2-the-extreme-importance-of-pc-board-stack-up',
+            'info_dict': {
+                'id': 'rGGKQEAu8X-APkU68K8U_w',
+                'display_id': '3225198',
+                'ext': 'mp4',
+                'title': 'The Extreme Importance of PC Board Stack Up',
+                'thumbnail': 'https://cdn.vidyard.com/thumbnails/73_Q3_hBexWX7Og1sae6cg/9998fa4faec921439e2c04_small.jpg',
+                'duration': 3422,
+            },
+        },
+    ]
+
```

Comment:
If applying the suggestion above:
```suggestion
    @property
    def _HEADERS(self):
        return {'referer': 'https://play.vidyard.com/'}

```
Traditionally this has been written as `Referer` but perhaps the new header processing doesn't care?

Reply:
Referrer handling extracted as suggested as of b8345dd2076859b85857a82e010d4b793978b041.

Reply:
Header processing doesn't seem to care - it works just fine, and lower case is used in a number of other extractors